packages/eslint-plugin-next/lib/rules/no-script-in-head.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow using <title> with Head from next/document',
```

Comment:
i think this is left over from copypaste?

Reply:
Yes, just waiting for a response for the [failing test](https://github.com/vercel/next.js/blob/canary/test/integration/script-loader/pages/_document.js) to do more work

Reply:
hmm, i guess the quick solution would be to add
```js
  eslint: {
    ignoreDuringBuilds: true,
  },
```
to
https://github.com/vercel/next.js/blob/canary/test/integration/script-loader/next.config.js

otoh - not sure if [this test](https://github.com/vercel/next.js/blob/dd7a54c5099c781273b01f97788e980fd203e188/test/integration/script-loader/pages/_document.js#L4) is actually correct?

Reply:
@awareness481 could you try to rebase - i think the issue with the failing test should be fixed with #27626

Reply:
Yep, I'll get on it (along with the request changes) within the week. Cheers for un-blocking the PR :blush: